client/src/index.jsx

```diff
@@ -173,7 +191,43 @@ class App extends React.Component {
 		}));
 	}
 
+	renderPreviousPage() {
+		if (this.state.page > 0) {
+			this.setState({
+				page: this.state.page -= 1
+			});
+		} else {
+			console.log('Already on the first page');
+		}
+		this.viewHomepage(); // this should be renamed or a new function created if we want a button to reset state.page = 0
+	}
+
+	renderNextPage() {
+		// Quality checks for too many pages?
+		console.log('loading next page');
+		//const nextPage = this.state.page + 1;
+		this.setState({
+			page: this.state.page += 1
+		});
+		this.viewHomepage();
+	}
+
 	render() {
+		let backButton;
+		if (this.state.page === 0) {
+			backButton = null;
```

Comment:
Stylistic suggestion: don't make the buttons go away, just make the disabled. its jarring to have the screen change and buttons move around